backend/engine/plugins/trivy_sbom/parser.py

```diff
@@ -1,13 +1,10 @@
 """
 trivy output parser
 """
-import json
-from engine.plugins.lib import utils
-
-logger = utils.setup_logging("trivy SBOM")
-
-import json
+from engine.plugins.lib.utils import setup_logging
+from engine.plugins.lib.utils import convert_type
 
+logger = setup_logging("trivy SBOM")
```

Comment:
I think we should probably have this one match the format of the others was well (lowercase, underscores instead of spaces, etc). Apologies if I missed this one on first review.